vault/resource_aws_auth_backend_client.go

```diff
@@ -5,6 +5,7 @@ package vault
 
 import (
 	"context"
+	automatedrotationutil "github.com/hashicorp/terraform-provider-vault/internal/rotation"
```

Comment:
maybe it doesn't apply to tfvp, but _occasionally_ some linter somewhere will complain to me about this sort of import order.